packages/design-picker/src/components/design-picker-category-filter/style.scss

```diff
@@ -5,6 +5,41 @@
 $design-picker-category-filter-text-color: #3c434a;
 $design-picker-category-filter-text-color-active: var(--studio-gray-100);
 
+.design-picker {
+	.design-picker__filters {
+		display: flex;
+		justify-content: space-between;
```

Comment:
I don't think these 2 rules should be part of `design-picker-category-filter/style.scss` since that class it not rendered by any component in `design-picker-category-filter` but by `unified-design-picker.tsx`.

Reply:
Same for the styles of the `design-picker__category-filter`, `design-picker__design-your-own-button`, and `design-picker__design-your-own-button-without-categories` classes.

Reply:
I'll move them back except for the `design-picker__category-filter` which should live in the Category Filter file.

Reply:
> the design-picker__category-filter which should live in the Category Filter file.

The class should be added by `packages/design-picker/src/components/design-picker-category-filter/unified-design-picker-category-filter.tsx` then, not by `packages/design-picker/src/components/unified-design-picker.tsx`.